phantomstubs/phantom/action_result.py

```diff
@@ -0,0 +1,9 @@
+class ActionResult:
```

Comment:
I am guessing more is coming here regarding this?
A doc str would ease my newbie curiosity

Reply:
Oh, are we to override this?
If that is the case, doc strings like `"""Override"""` are nice

Reply:
everything in the `phantomstubs` folder is just dummy implementations of stuff in the `phantom` namespace, because the `phantom` lib is only available inside of a running phantom instance. It's a cheap way to be able to mock it.

Reply:
I'll add a note on this either to Contributing.md or as a docstring